Automattic/jetpack
# Remove style from calypsoify

Fixes:
https://github.com/Automattic/wp-calypso/issues/73287

## Proposed changes:

This PR removes styles from the Calypsoify module in the Jetpack plugin. This is to align styles that are applied specifically for Calypso with the styles that are applied on WordPresss.com. 

Calypsoify was intended to create visual unification between wp-admin and Calypso but it is causing issues and the goal is to eventually phase it out completely. See p1693905134182179-slack-CDLH4C1UZ

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Via wp-admin:

* Build the plugin and go to your local development environment
* Edit a post or page, eg: http://localhost/wp-admin/post.php?post=1&action=edit&calypsoify=1 (ensure that calypsoify=1 is added as query param).
* Ensure that the button colors are now the standard colors and not the colors of the default calypso theme (`#c9356e`)

Via wpcom:

* Via Jetpack Beta, pick the remove-style-from-calypsoify for a WoA site.
*  Go to pages and edit a page (the editor should be iframed via Calypso)
* The buttons in the top bar should now show the default colors (`#096484`)
